lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Transaction.kt

```diff
@@ -210,11 +210,6 @@ data class InvokeTransactionV3(
             senderAddress = senderAddress,
             version = version,
             resourceBounds = resourceBounds,
-            tip = tip,
-            paymasterData = paymasterData,
-            accountDeploymentData = accountDeploymentData,
-            nonceDataAvailabilityMode = nonceDataAvailabilityMode,
-            feeDataAvailabilityMode = feeDataAvailabilityMode,
```

Comment:
Should we also make primary constructor private similarly to `TransactionPayload`, v3 params and so on, and create constructor with hardcoded fields? I decided to not do it for now:
1. to not further clutter the pR
2. because there were previously some non-rpc fields and we didn't hardcode anything
3. we mainly use `TransactionFactory` to correctly create `Transaction` objects anyway

This would be easily solved with #394, but we decided to do it separately.

Reply:
no need, I suppose